docs/index.md

```diff
@@ -22,6 +22,8 @@ antsibull-changelog is covered by the [Ansible Code of Conduct](https://docs.ans
 To see how antsibull-changelog changes, please look at the
 [antsibull-changelog changelog](https://github.com/ansible-community/antsibull-changelog/blob/main/CHANGELOG.md).
 
+> Need help or want to discuss the project? See our [Community guide](community.md) to learn how to join the conversation!
+
```

Comment:
Is this really how you want this to look?
![image](https://github.com/user-attachments/assets/22431550-9e4a-46d1-8e87-9ffc6098b95f)

If not, remove the `>` above.

Reply:
Alternatively, you can make it a proper note:
```md
!!! note
    Need help or want to discuss the project? See our [Community guide](community.md) to learn how to join the conversation!
```

Reply:
The result looks like this:
![image](https://github.com/user-attachments/assets/15d1ee42-6b2c-4cf2-bc3b-8933eefa5ff6)

Reply:
LGTM, thanks! didn't know that